docs/deployment/gardenlet_api_access.md

```diff
@@ -62,7 +62,47 @@ Today, the following rules are implemented:
 
 The `SeedAuthorizer` is implemented as [Kubernetes authorization webhook](https://kubernetes.io/docs/reference/access-authn-authz/webhook/) and part of the [`gardener-admission-controller`](../concepts/admission-controller.md) component running in the garden cluster.
 
-⚠️ This authorization plugin is still in development and should not be used yet.
+🎛 In order to activate it, you have to follow these steps:
+
+1. Set the following flags for the `kube-apiserver` of the garden cluster (i.e., the `kube-apiserver` whose API is extended by Gardener):
+   - `--authorization-mode=RBAC,Node,Webhook` (please note that `Webhook` should be the last one in the list; `Node` might not be needed if you use a virtual garden cluster)
```

Comment:
> please note that `Webhook` should be the last one in the list

Can you elaborate on why that is?

Reply:
It's due to the "dependency cycle" that you mentioned as well some weeks ago: KAPI is depending on GAC, GAC is depending on GAPI, and GAPI is depending on KAPI.
When I was testing this out locally I haven't observed any issues, but during my tests on a real landscape, I experienced the situation that GAPI couldn't start because KAPI tried to authenticate its requests with GAC, but GAC wasn't running for obvious reasons.
I'm not entirely sure what was different compared to my local environment, but I figured that having `Webhook` as the last auth plugin is also better for another reason: All users (except gardenlets) are authenticated via RBAC anyways, so when their requests are allowed then the webhook is not even called. For gardenlets, there won't be any RBAC rules, so the webhook gets called eventually.

Reply:
Ok, that makes perfect sense. Thanks for clarifying.
Would you care to add a short sentence as an explanation to the doc?